heclib/javaHeclib/CMakeLists.txt

```
@@ -6,7 +6,9 @@ if(JNI_FOUND)
     if( ${CMAKE_C_COMPILER_ID} MATCHES GNU)    
         target_compile_options(javaHeclib PRIVATE -fPIC -std=gnu99)
     elseif(${CMAKE_C_COMPILER_ID} MATCHES SunPro)
-        target_compiler_options(javaHecib PRIVATE -m64 -xmemalign=2i -xcode=pic32 -c -Isrc/headers -mt -xcode=abs64 -xstrconst -xc99=all -xCC -g -xs)
+        target_compile_options(javaHecib PRIVATE -m64 -xmemalign=2i -xcode=pic32 -c -Isrc/headers -mt -xcode=abs64 -xstrconst -xc99=all -xCC -g -xs)
```

```suggestion
        target_compile_options(javaHeclib PRIVATE -m64 -xmemalign=2i -xcode=pic32 -c -Isrc/headers -mt -xcode=abs64 -xstrconst -xc99=all -xCC -g -xs)
```

It appears there was another typo in that section.

---

The source for heclib_c directory was being copied into each of the test projects, making it very STATIC , but slowing down builds also.

---

About that powf_wrap issue:   I'm considering not worrying about older Glib versions. (centos 7, older ubuntu) and just drop this wrapper:

https://github.com/HydrologicEngineeringCenter/hec-dss/blob/main/heclib/heclib_c/src/Internal/powf_wrap.c